TiagoGIM/jack-pot-api
# (FEAT) : Authentication, Ticket CRUD, User Management, and Bets API

# Feature Implementation: Authentication, Ticket CRUD, User Management, and Bets API

## Description

This PR introduces several features to enhance the functionality of our backend. Here is a summary of the implemented features:

### 1. Authentication with JWT

Implemented authentication using JWT (JSON Web Tokens) strategy to secure our endpoints.

### 2. Ticket CRUD Operations

Added CRUD (Create, Read, Update, Delete) operations for managing tickets. Users can now create and edit tickets.

### 3. User Management API

Introduced endpoints for creating new users. Now users can be registered using the backend API.

### 4. Bets API

Implemented endpoints to retrieve and update bets. Users can now interact with their bets through the API.

## Changes Made

- Authentication with JWT strategy
- CRUD operations for tickets
- User management endpoints
- Bets API

## How to Test

1. Make sure the backend server is running.
2. Use API endpoints for authentication, ticket management, user creation, and bets.
